test/unit/firestore/firestore.spec.ts

```diff
@@ -98,23 +98,25 @@ describe('Firestore', () => {
       it(`should throw given invalid app: ${ JSON.stringify(invalidApp) }`, () => {
         expect(() => {
           const firestoreAny: any = FirestoreService;
-          return new firestoreAny(invalidApp);
+          const firestoreService: FirestoreService = new firestoreAny(invalidApp);
+          return firestoreService.getDatabase('(default)')
```

Comment:
Should we make this `'(default)'` a constant?

Reply:
Imported from Server SDK